docs/examples/dl3_tool_config_srcdep.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
You do not need to add separate config files to just add alpha_cut. you can just add to the existing ones, as the Tool will use the values as required by the particular functions or usage.